packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -244,6 +263,14 @@ export default defineComponent({
     },
 
     canCreatePublicLinks() {
+      const reSharingEnabled = this.capabilities?.files_sharing?.resharing === true
```

Comment:
Please use `this.hasResharing` (see comment in setup function for the capability composable).